service/server.py

```diff
@@ -292,6 +312,19 @@ def format_proprietors(proprietors_data):
     return formatted_proprietors
 
 
+def format_pi_name(name):
+    name_list = []
+    if 'title' in name:
+        name_list.append(name['title'])
+    if 'forename' in name:
+        name_list.append(name['forename'])
+    if 'surname' in name:
+        name_list.append(name['surname'])
+    if 'decoration' in name:
+        name_list.append(', ' + name['decoration'])
+    return ' '.join(name_list)
```

Comment:
I think this would give you `Mr Joe Bloggs , MBA` instead of what we probably want which is `Mr Joe Bloggs, MBA`